geoportailv3/static/js/mymapsservice.js

```diff
@@ -470,9 +470,8 @@ app.Mymaps.prototype.loadFeatures_ = function() {
 
 /**
  * update the map with layers
- * @private
  */
```

Comment:
okay then, I thought in the past that we would standardise on that behaviour, but guess didnt understand it properly. https://github.com/Geoportail-Luxembourg/geoportailv3/issues/865